web-server/plugins/slycat-dac/js/dac-scatter-buttons.js

```diff
@@ -694,4 +718,76 @@ module.get_color_by_col = function()
     return curr_color_by_col;
 }
 
+// lof button calls server to recompute lof then draws new color
+function lof_button ()
+{
+
+    // call server to compute lof
+	client.post_sensitive_model_command(
+    {
+        mid: mid,
+        type: "DAC",
+        command: "compute_lof",
+        parameters: {alpha: alpha_sliders.get_alpha_values(),
+                     subset: selections.get_subset(),
+                     include_columns: var_include_columns,
+                     use_PCA_comps: use_PCA_comps},
+        success: function (result)
+            {
+
+                lof_color_by_col = JSON.parse(result)["lof"];
```

Comment:
before you run this block of code you probably want to check if result returned the correct status (like 200).  You would probably also want to check that the key is in the json that is returned with something like JSON.parse(result).hasOwnProperty("lof")